codespell_lib/data/dictionary.txt

```diff
@@ -13111,6 +13123,7 @@ skiping->skipping
 skipp->skip, skipped,
 skippd->skipped
 skipt->skip
```

Comment:
Not you, but this should probably suggest skipped too.

Reply:
done